packages/cbioportal-utils/src/mutation/MutationUtils.ts

```diff
@@ -89,3 +93,13 @@ export function uniqueGenomicLocations(
 
     return _.values(genomicLocationMap);
 }
+
+export function normalizeChromosome(chromosome: string) {
```

Comment:
Apparently this is duplicate of this one: https://github.com/cBioPortal/cbioportal-frontend/blob/9bf8a68e8ad0b36d75da3a3b47bd4eeeff3dd50e/src/shared/lib/IGVUtils.ts#L93

Maybe move the already existing function here (I mean just copy the old implementation from `IGVUtils`, and delete the one there), and import the new function inside `IGVUtils.ts`.